debian/rules

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/make -f
 # -*- makefile -*-
 
-DEBIAN_DIST := $(shell lsb_release -ds | tr -d '()' | sed -e 's/\#/ /g')
+DEBIAN_DIST = "$(shell dpkg-vendor --query vendor)"
```

Comment:
The `:=` form is nice because it means make will only evaluate the variable once, which is good in this case since it Diane a process who's output will not change.

Reply:
Also, the outer quotes don't really matter to make. The right hand side of taken verbatim. I don't think it hurts, but they become part of the value.

Reply:
Interesting, updated.

Reply:
Yes, for the outer quotes - thing is I literally copied this handling from https://salsa.debian.org/gstreamer-team/gst-plugins-bad1.0/-/blob/master/debian/rules and didn't think much of it :-D. Let's leave it at that.